cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/JaasConfig.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.operator.assembly;
+
+import java.util.Map;
+import java.util.Objects;
+import java.util.StringJoiner;
+
+/**
+ * JaasConfig is a Java class that provides a utility method for generating Java Authentication and
+ * Authorization Service (JAAS) configuration strings.
+ */
+public class JaasConfig {
```

Comment:
Is a new class really necessary, or could we put this code in `AuthenticationUtils`?